scripts/print-warnings/print-warnings.js

```diff
@@ -6,20 +6,16 @@
  */
 'use strict';
 
-const babylon = require('babylon');
+const parser = require('@babel/parser');
 const fs = require('fs');
 const through = require('through2');
 const traverse = require('babel-traverse').default;
 const gs = require('glob-stream');
 
 const evalToString = require('../shared/evalToString');
 
-const babylonOptions = {
+const parserOptions = {
   sourceType: 'module',
```

Comment:
`trailingFunctionCommas` should be removed from here, see [babel's changelog](https://github.com/babel/babel/blob/eac4c5bc17133c2857f2c94c1a6a8643e3b547a7/packages/babel-parser/CHANGELOG.md#691-2016-08-23).